IPython/sphinxext/ipython_directive.py

```diff
@@ -138,7 +138,13 @@
 
 # Third-party
 from docutils.parsers.rst import directives
-from sphinx.util.compat import Directive
+import sphinx
+# sphinx.version_info available since 1.2, and we require 1.3
+# sphinx.util.compat deprecated at 1.6 and removed at 1.7
+if sphinx.version_info < (1, 5, 7):
+    from sphinx.util.compat import Directive
+else:
+    from docutils.parsers.rst import Directive
```

Comment:
I find it in docutils-0.5 and in docutils-0.6 it is declared as inheriting from ``object`` (earlier it was just ``class Directive``).

Anyway, Sphinx 1.3 has ``docutils>=0.11`` in its `setup.py` requirements. And I think IPython currently requires ``Sphinx>=1.3``.

However I admit I am not familiar with why there was a wrapper to Directive in sphinx.util.compat before 1.6, I will check a bit.